sdk-java/src/main/java/io/littlehorse/sdk/common/config/LHConfig.java

```diff
@@ -403,6 +403,6 @@ protected String[] getEnvKeyPrefixes() {
      * @return the number of worker threads to run.
      */
     public int getWorkerThreads() {
-        return Integer.valueOf(getOrSetDefault(NUM_WORKER_THREADS_KEY, "8"));
+        return Integer.valueOf(getOrSetDefault(NUM_WORKER_THREADS_KEY, "2"));
```

Comment:
Can you do one of two things:

1. Update the other SDK's and update the docs with the new default value, OR
2. Leave this at `8`